tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -773,7 +773,11 @@ def pack_params_to_args_namespace(**kwargs):
     for arg in vars(argv):
```

Comment:
can you please add more description what `non_default_params` is?

Reply:
Added comment about `non_default_params`